src/Ting/Cache/Cache.php

```diff
@@ -86,6 +86,8 @@ protected function stopLog($miss = false)
 
     /**
      * {@inheritdoc}
+     *
+     * @return bool
```

Comment:
To avoid BC breaks but still fix the deprecations, I added the `@return` annotation